src/main/pg/rx.c

```diff
@@ -66,9 +66,9 @@ void pgResetFn_rxConfig(rxConfig_t *rxConfig)
         .rc_smoothing_input_cutoff = 0,      // automatically calculate the cutoff by default
         .rc_smoothing_derivative_cutoff = 0, // automatically calculate the cutoff by default
         .rc_smoothing_debug_axis = ROLL,     // default to debug logging for the roll axis
-        .rc_smoothing_input_type = RC_SMOOTHING_INPUT_BIQUAD,
-        .rc_smoothing_derivative_type = RC_SMOOTHING_DERIVATIVE_AUTO, // automatically choose type based on feedforward method
-        .rc_smoothing_auto_factor = 10,
+        .rc_smoothing_input_type = RC_SMOOTHING_INPUT_PT3,
+        .rc_smoothing_derivative_type = RC_SMOOTHING_INPUT_PT3,
```

Comment:
This will default derivative to PT2, is it what you want or its only typo instead of correct RC_SMOOTHING_DERIVATIVE_PT3 enum ?

Reply:
oh gosh thank you!  That was a typo - so glad you found it!
fixed.